.github/workflows/reuse.linux.yml

```diff
@@ -72,6 +72,26 @@ jobs:
           target: ${{ matrix.arch }}-unknown-linux-${{ matrix.libc }}
           artifacts: all
 
+  build_address_sanitizer:
+    name: build_${{ matrix.arch }}_${{ matrix.libc }}_asan
+    runs-on: [ self-hosted, X64, Linux, 16c32g, "${{ inputs.runner_provider }}" ]
```

Comment:
@everpcpc do we need to downgrade `16c32g` to small to save billing? This build not affect the following steps.

Reply:
Yes, if 8c16g is enough to build.